src/common/translation.cpp

```diff
@@ -248,9 +296,9 @@ wxString GetPreferredUILanguage(const wxArrayString& available)
 
 #else
 
-// On Unix, there's just one language=locale setting, so we should always
-// use that.
-#define GetPreferredUILanguage GetPreferredUILanguageFallback
+// On *nix, a list of preferred languages may be specified in the LANGUAGE
+// environment variable.
+#define GetPreferredUILanguage GetPreferredUILanguageFromEnvVariable
```

Comment:
The whole `GetPreferredUILanguageFromEnvVariable()` function should be just called `GetPreferredUILanguage()` and its body moved to here, getting rid of the `#define` altogether. Its name is misleading anyway (the *Fallback one uses environment variable too) and it is only used on UNIX, so it doesn't make sense to compile it on other platforms and to obfuscate the code with such renamings.